test/integration/test-amp-ad-3p.js

```diff
@@ -24,7 +24,8 @@ function createFixture() {
   return createFixtureIframe('test/fixtures/3p-ad.html', 3000, () => {});
 }
 
-describe('amp-ad 3P', () => {
+//TODO(erwinm, #30528): unskip this for IE when possible
+describe.configure().run('amp-ad 3P', () => {
```

Comment:
IIRC, `describe` and `describes` were recently fixed to reliably not run IE tests by default, so you don't have to adopt workaround.

/cc @rcebulko who knows the status of this.

Reply:
`describes` skips IE by default, but the native Mocha `describe` does not. This code is correct as written.

Reply:
so it was running on the IE tests earlier and failing on this one test with a normal `describe` block. @rcebulko lmk if i need to change it to anything

Reply:
@erwinmombay This code is correct as you have it written. Your above comment/assessment is accurate.